### PR TITLE
Feature/cleanup actuator application yaml

### PR DESCRIPTION
**Pull request by kilbphilippSAP**
## Checklist

* [x] Test your changes as thoroughly as possible before you commit them. Preferably, automate your test by unit/integration tests.
* [x] Make sure `mvn -P integration-tests clean verify` runs for the whole project and, if you touched any code for a service in the `services` folder, ensure it can be run with `spring-boot:run`

## Description

- Fix actuator/health route in submission service
- cleanup application.yaml for callback & submission

### CODE REVIEW

**Review by hilmarf — APPROVED** [review]

File: services/submission/src/main/resources/application.yaml [hunk 1]
```diff
@@ -77,34 +77,27 @@ spring:
 # Actuator configuration
 management:
   server:
+    ssl:
+      enabled: false
     port: 8081
```
Comment (kilbphilippSAP) [inline services/submission/src/main/resources/application.yaml:82]: @hilmarf @ioangut Do we want the actuator on a separate port or on the same as the service itself?

Comment (hilmarf) [reply services/submission/src/main/resources/application.yaml:82]: @johanneseschrig - any best practices? How have you come to that port?

**Review by sorin-iovita — APPROVED** [review]

**Comment by hilmarf** [comment]
@kilbphilippSAP / @mibrasap - this is labeled with 'do not merge', but I think some of the changes are still relevant, aren't they?

File: services/submission/src/main/resources/application.yaml (hunk 1 above)
Comment (hilmarf) [reply services/submission/src/main/resources/application.yaml]: port: should be 8081 which shouldn't be accessible from outside

**Review by hilmarf — APPROVED** [review]

**Pull request merged**